main/HazardousBody.md

```diff
@@ -54,9 +55,10 @@ Body
 
 |Property|Format|Description|
 |--------|------|-----------|
-|heat|Double|The average heat on the body.|
-|interval|Single|How much time passes between applying the heat to a vessel.|
-|AltitudeCurve|FloatCurve|A multiplier of the average heat that gets applied at a certain altitude.|
-|LatitudeCurve|FloatCurve|A multiplier of the average heat that gets applied at a certain latitude.|
-|LongitudeCurve|FloatCurve|A multiplier of the average heat that gets applied at a certain longitude.|
-|HeatMap|File Path|A greyscale map for fine control of the heat on the body. Black = 0, White = 1.|
+|sumTemp|Boolean|LOOK AT CODE/ASK SIGMA|
```

Comment:
What? Will this be filled out or just as a placeholder?

Reply:
Yes that will be filled out, It is `Whether or not to add the ambientTemp to the calculated temperature.`

Reply:
Are you changing it or should I?